ably-ios/ARTDefault.h

```diff
@@ -14,8 +14,19 @@
 + (NSString*)restHost;
 + (NSString*)realtimeHost;
 + (int)TLSPort;
-+ (NSTimeInterval)connectTimeout;
-+ (NSTimeInterval)disconnectTimeout;
-+ (NSTimeInterval)suspendTimeout;
+
++ (NSTimeInterval)connectTimeout; //Realtime
+
+/**
+ When the client is in the DISCONNECTED state, once this TTL has passed, the client should change the state to the SUSPENDED state signifying that the state is now lost i.e. channels need to be reattached manually.
+ 
+ Note that this default is override by any connectionStateTtl specified in the ConnectionDetails of the CONNECTED ProtocolMessage.
```

Comment:
Yes, this is true, but not yet implemented in any client libraries so does not need to be implemented